### PR TITLE
Rewrite configuration to irc3-style

### LINKED ISSUE #148: Rewrite Plugin configs to irc3-style config.ini
Existing plugin configuration should be converted to the ini-style config irc3 uses natively. 

An example plugin configuration could look like:

``` ini
[plugins.pluginname]
option1 = true

option2 = Lorem Ipsum

option3 =
    1
    2
    3

option4 = 12345
```

You can access those options from the code via the following code

``` python
# Set the default options
config = {
    option1 = False,
    option2 = "Lorem Ipsum",
    option3 = [1, 2, 3, 4, 5],
    option4 = 12345
}
config.update(self.bot.config.get(__name__, {}))

# You can access an option like this
option1 = config['option1']
```

### PR DESCRIPTION
**Pull request by mape2k**
Fixes #148

### CODE REVIEW

File: config.ini.example [hunk 1]
```diff
@@ -59,6 +67,70 @@ mkzero!*@* = all_permissions
...
+SpacestatusCommits =
+    https://github.com/Bytespeicher/space-status/commits/master.atom
+    /tmp/feed_spacestatus.cache
+    github
+
+[plugins.shorturl]
+clarifai_app_id = yourappid
+clarifai_app_secret = yourappsecret
+
+[plugins.status]
```
Comment (mkzero) [inline config.ini.example]: shouldn't that be a more global option, as its not a plugin option per se but rather an option for a library function?

Comment (mape2k) [inline config.ini.example]: Maybe [lib.status] is more intuitive and works? not tested yet...

Comment (mkzero) [inline config.ini.example]: +1 for that if it works. Should work though, it's just another name for the section we're using. Everything else looks fine to me.

Comment (mape2k) [inline config.ini.example]: Moved to lib.spaceapi as lib is named spaceapi...

**Pull request merged**